client/src/components/Footer/style.js

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+export const Container = styled('div')`
+  padding: ${({ theme }) => `${theme.spacings[3]} ${theme.spacings[5]}`};
+  width: 100%;
+  height: 110px;
+  background: white;
+`;
```

Comment:
```suggestion
  a {
    display: block;
  }
`;
```
you can select the child and give them style better than inline style and more cleaner